README.md

```diff
@@ -7,10 +7,23 @@ CWQ is designed for fast searching and presentation of log data in multiple form
 The basic command structure is:
 
 ```bash
-cwq --log-group <log-group> '<query>'
+cwq --log-group <log-group>
 ```
 
-Where `<log-group>` is the log group you want to search and `<query>` is the [CloudWatch Logs Insights query](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_QuerySyntax.html) to search with.
```

Comment:
Per above, I think this line can stay the same as well.

Reply:
Only use lines 15 - 20: 06a25cf7cb31bb0c8129ee1e7ac3b4fde27f3895.